src/backend/expungeservice/expunger.py

```diff
@@ -60,6 +60,14 @@ def run(self) -> bool:
                     (charge.disposition.date + relativedelta(years=1), "Time-ineligible under 137.225(1)(b)")
                 )
 
+            if charge.convicted() and charge.case()().get_probation_revoked():
+                eligibility_dates.append(
+                    (
+                        charge.disposition.date + relativedelta(years=10),
```

Comment:
Why does this use conviction date instead of the parsed revocation date?

Relatedly, looking at the fixtures and this parser code it's not clear where revoked probation date actually comes from in the case detail page. Can only a single revocation apply to an entire case? In which case the eligibility dates of all convictions would be the same and the extra reason comment below is not applicable